ReportSimulationOutput/measure.rb

```diff
@@ -1848,16 +1874,17 @@ def get_report_meter_data_annual(meter_names, unit_conv = UnitConversions.conver
     return val
   end
 
-  def get_report_variable_data_annual(key_values, variables, unit_conv = UnitConversions.convert(1.0, 'J', 'MBtu'), is_negative: false)
+  def get_report_variable_data_annual(key_values, variables, unit_conv = UnitConversions.convert(1.0, 'J', 'MBtu'), is_negative: false, is_storage: false)
     return 0.0 if variables.empty?
 
     neg = is_negative ? -1.0 : 1.0
+    sto = is_storage ? -1.0 : 1.0
```

Comment:
A bit silly to create a new argument that behaves exactly the same as the `is_negative` argument. When calling this method, just use `is_negative: (end_use.is_negative || end_use.is_storage)`